ARK Server Manager/ServerSettingsControl.xaml

```diff
@@ -1165,7 +1170,12 @@
                     </Expander>
                     <Expander IsExpanded="True" Style="{DynamicResource ExpanderStyle1}">
                         <Expander.Header>
-                            <TextBlock HorizontalAlignment="Left" TextWrapping="Wrap" Text="{DynamicResource ServerSettings_HUDAndVisualsLabel}" VerticalAlignment="Center" FontWeight="Bold" FontSize="13.333"/>
+                            <StackPanel Orientation="Horizontal">
+                                <TextBlock HorizontalAlignment="Left" TextWrapping="Wrap" Text="{DynamicResource ServerSettings_HUDAndVisualsLabel}" VerticalAlignment="Center" FontWeight="Bold" FontSize="13.333"/>
+                                <Button Click="HUDAndVisuals_Reset" Width="22" Height="22" ToolTip="{DynamicResource ServerSettings_ResetSectionTooltip}" Margin="20,0,0,0">
```

Comment:
Check out the latest commit: 142d49d9b280247eadc0411a84518e9c85400026

That replaces the Save button's Save_Click event handler with a very similar Command implementation.  I also added a few notes.  Let me know if you have any more questions.